common/file/UtilTest.cpp

```diff
@@ -20,26 +20,36 @@
 
 #include <cppunit/extensions/HelperMacros.h>
 #include <string>
+#include <vector>
 
 #include "ola/file/Util.h"
 #include "ola/testing/TestUtils.h"
 
+#ifdef _WIN32
+const char PATH_SEPARATOR = '\\';
+#else
+const char PATH_SEPARATOR = '/';
+#endif  // _WIN32
+
```

Comment:
Can we not use the definition that's already in https://github.com/OpenLightingProject/ola/blob/0.10/common/file/Util.cpp#L47-L51

Then perhaps add a test to ensure it's been set correctly?